objects/actors/player/Player.tscn

```diff
@@ -184,3 +188,11 @@ shape = SubResource("CircleShape2D_rqsaa")
 [node name="AttackTimer" type="Timer" parent="." index="3"]
 unique_name_in_owner = true
 one_shot = true
+
+[node name="Collector" parent="." index="4" instance=ExtResource("5_q1uwo")]
+collision_mask = 4
```

Comment:
There is a weird thing with `Collector` and its `AudioPlayer`, I think we should remove the second one from its parent as we don't use it yet.